spring-cloud-netflix-eureka-client/src/test/java/org/springframework/cloud/netflix/eureka/http/RestTemplateTransportClientFactoryTest.java

```diff
@@ -26,31 +26,31 @@
  */
 class RestTemplateTransportClientFactoryTest {
 
-	private RestTemplateTransportClientFactory transportClientFatory;
+	private RestTemplateTransportClientFactory transportClientFactory;
 
 	@BeforeEach
 	void setup() {
-		transportClientFatory = new RestTemplateTransportClientFactory();
+		transportClientFactory = new RestTemplateTransportClientFactory();
 	}
 
 	@Test
 	void testWithoutUserInfo() {
-		transportClientFatory.newClient(new DefaultEndpoint("http://localhost:8761"));
+		transportClientFactory.newClient(new DefaultEndpoint("http://localhost:8761"));
```

Comment:
I know this code was already here, but as you're changing this area, can you please add appropriate assertions to all the test methods? and remove `test` prefix from the test method names?